lib/router.js

```diff
@@ -633,3 +634,38 @@ Router.prototype.toString = function toString() {
 
     return (str);
 };
+
+
+/**
+ * Return information about the routes registered in the router.
+ * @public
+ * @returns {object} The routes in the router.
+ */
+Router.prototype.getRoutes = function getRoutes() {
```

Comment:
This seems misnamed - it's not actually returning the routes, just a bucket of info about the routes. I remember we were thinking about implementing this as `toJSON()`?